chandra_aca/tests/test_aca_image.py

```diff
@@ -423,7 +422,7 @@ def test_aca_image_operators():
 def test_flicker_numba():
     a = ACAImage(np.linspace(0, 800, 9).reshape(3, 3))
     a.flicker_init(flicker_mean_time=1000, flicker_scale=1.5, seed=10)
-    for ii in range(10):
+    for _ii in range(10):
```

Comment:
`_` is the preferred variable for an unused loop parameter. You don't need to bother changing this in test code but keep it in mind for the future.

Reply:
`_` behaves like any other variable to Python and pytest. It is just humans and linters that give it a special meaning.